DEVELOPING.md

```diff
@@ -179,72 +192,282 @@ kubectl get pod -n ambassador
 
 🥳 If all has gone well then you should have your development environment setup for building and testing Emissary-ingress.
 
-### What's Next?
+### Step 7: Dev-loop
+
+Now that you are all setup and able to deploy a development container of Emissary-ingress to a cluster, it is time to start making some changes.
+
+Lookup an issue that you want to work on, assign it to yourself and if you have any questions feel free to ping us on slack in the #emissary-dev channel.
+
+Make a change to Emissary-ingress and when you want to test it in a live cluster just re-run
+
+`make deploy`
+
+This will:
+
+- recompile the go binary
+- rebuild containers
+- push them to the docker registry
+- rebuild helm charts and manifest
+- reapply manifest to cluster and re-deploy Emissary-ingress to the cluster
+
+> *Do I have to run the other make targets `make image` or `make push` ?*
```

Comment:
Don't think make image and make push was mentioned beforehand. Maybe just reword to say these targets exists if you want to do individual things